src/components/templates/Camp/components/Map.scss

```diff
@@ -65,7 +64,6 @@ $border-radius: 28px;
     position: relative;
     width: 100%;
     height: 100%;
-    box-shadow: 0 4px 6px 0 rgba(0, 0, 0, 0.2);
```

Comment:
whoops these are unrelated fixes for camp venue where it had some unnneeded shadows.